.travis.yml

```diff
@@ -0,0 +1,33 @@
+language: c
+
+cache:
+  - ccache
+
+os: linux
```

Comment:
We definitively want to test windows etc...

Here is what Vulkan-Loader has : 

```
matrix:
  # Show final status immediately if a test fails.                                                
  fast_finish: true
  allow_failures:
    - env: CHECK_COMMIT_FORMAT=ON
    - env: VULKAN_BUILD_TARGET=GN
  include:
    # Linux GCC debug build.                                                                      
    - os: linux
      compiler: gcc
      env: VULKAN_BUILD_TARGET=LINUX
    # Linux clang debug build.                                                                    
    - os: linux
      compiler: clang
      env: VULKAN_BUILD_TARGET=LINUX
    # Linux GN debug build                                                                        
    - os: linux
      env: VULKAN_BUILD_TARGET=GN
    # MacOS clang debug build.                                                                    
    - os: osx
      compiler: clang
      env: VULKAN_BUILD_TARGET=MACOS
    # Check for proper clang formatting in the pull request.                                      
    - env: CHECK_FORMAT=ON
    # Check for proper commit message formatting for commits in PR                                
    - env: CHECK_COMMIT_FORMAT=ON
```

Reply:
Ah, I didn't think to look at what other KHR projects are doing, I just wrote it from scratch :upside_down_face:

I'll rewrite it to match the Loader travis config :+1:

Reply:
Yeah, I think we should probably add the following files : 
```
scripts/update_deps.py
scripts/known_good.json
```
So the right headers are downloaded.